src/unofficial.md

```diff
@@ -15,6 +15,7 @@ Application domains:
 * [A thoughtful introduction to the pest parser](https://pest.rs/book/)
 * [Rust and WebAssembly](https://rustwasm.github.io/docs/book/)
 * [The `wasm-bindgen` Guide](https://rustwasm.github.io/docs/wasm-bindgen/)
+* [The `wasm-pack` Docs](https://rustwasm.github.io/docs/wasm-pack/)
```

Comment:
As I mention in the README, my convention is to use the literal title of the book in question:
```suggestion
* [Hello wasm-pack!](https://rustwasm.github.io/docs/wasm-pack/)
```